lib/hanami/view/template.rb

```diff
@@ -7,9 +7,17 @@ module View
     # @since 0.1.0
     class Template
       def initialize(template, encoding = Encoding::UTF_8)
-        # NOTE disable_escape: true is for Slim compatibility
-        # See https://github.com/hanami/assets/issues/36
-        @_template = Tilt.new(template, nil, default_encoding: encoding, disable_escape: true)
+        default_options = {default_encoding: encoding}
+
+        options = default_options
```

Comment:
@nerdinand Why is this needed?

Reply:
I just wanted to make the distinction clearer that the `default_encoding` option is always being passed.